src/main/java/com/google/sps/servlets/MentorEvidenceServlet.java

```diff
@@ -0,0 +1,72 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.sps.classes.Utility;
+import java.io.IOException;
+import java.sql.*;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * 
+ */
+@WebServlet("/mentor-evidence")
+public class MentorEvidenceServlet extends HttpServlet {
+  
+  /**
+   * Receives evidence by a mentor to prove them as a previous intern and adds it to database.
+   */
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get variable from HTML form.
+    String paragraph = request.getParameter("paragraph");
+
+    // Insert mentor evidence to database.
+    addMentorEvidence(paragraph);
+    response.sendRedirect("/index.html");
+  }
+
+  /**
+   * Inserts evidence provided by mentor to MentorEvidence table in database.
+   */
+  private void addMentorEvidence(String paragraph) {
+    int userId = Utility.getUserId();
+
+    // Set up query to insert new experience tag to user.
+    String query = "REPLACE INTO MentorEvidence (mentor_id, paragraph) VALUES (?, ?)";
```

Comment:
How does the REPLACE work? Any reason to use it instead of INSERT? If so please add an implementation comment explaining.

Reply:
Replace works this way: if the value does not exist in the database, it creates a new one and inserts it, but if it does exist, it replaces it. It's in case the mentor wants to update the values he submits. Added the comment!